bitnami/fluentd/values.yaml

```diff
@@ -1019,16 +1192,7 @@ aggregator:
   ## @param aggregator.extraVolumeClaimTemplates Optionally specify extra list of additional volume claim templates for the Fluentd Aggregator pods in StatefulSet
   ##
   extraVolumeClaimTemplates: []
-## @param serviceAccount Pods Service Account. This top-level global entry is DEPRECATED. Please use "forwarder.serviceAccount" instead.
-## Only the forwarder was affected by the historical usage here.
-## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
-##
-serviceAccount: {}
-## @param rbac Role Based Access. This top-level global entry is DEPRECATED. Please use "forwarder.rbac" instead.
-## Only the forwarder was affected by the historical usage here.
-## ref: https://kubernetes.io/docs/admin/authorization/rbac/
-##
-rbac: {}
```

Comment:
Why do these parameters disappear?

Reply:
They were already deprecated and removed from the code. They have been for a long time under `forwarder.*`.

If set, they would trigger a warning (Check _helpers.tpl).

I thought about removing those messages too, but at least it doesn't make sense having them in the `values.yaml` in my opinion.

Reply:
Please go ahead and remove the warnings from the helpers too

Reply:
Done 👍